clustpy/deep/_train_utils.py

```diff
@@ -81,15 +93,14 @@ def get_trained_autoencoder(trainloader: torch.utils.data.DataLoader, optimizer_
         print("Autoencoder is not fitted yet, will be pretrained.")
         # Pretrain Autoencoder
         autoencoder.fit(n_epochs=n_epochs, optimizer_params=optimizer_params, dataloader=trainloader,
-                        device=device, optimizer_class=optimizer_class, loss_fn=loss_fn)
+                        optimizer_class=optimizer_class, loss_fn=loss_fn)
     if autoencoder.reusable:
         # If autoencoder is used by multiple deep clustering algorithms, create a deep copy of the object
         autoencoder = copy.deepcopy(autoencoder)
-
     return autoencoder
 
 
-def get_standard_initial_deep_clustering_setting(X: np.ndarray, n_clusters: int, batch_size: int,
+def get_standard_initial_deep_clustering_setting(X: np.ndarray | torch.Tensor, n_clusters: int, batch_size: int,
```

Comment:
maybe rename to `get_default_deep_clustering_setting`?

Reply:
renamed to get_default_deep_clustering_initialization